service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -129,11 +131,16 @@ public OrderTypeAndDocument buildOrderTypeAndDocument(OrderTypeAndDocument typeA
             .put("legalAdvisorName", JudgeAndLegalAdvisorHelper.getLegalAdvisorName(
                 caseData.getJudgeAndLegalAdvisor()))
             .put("children", getChildrenDetails(caseData))
+            .put("furtherDirections", getFurtherDirections(caseData.getOrderFurtherDirections()))
             .build();
 
         return orderTemplateBuilder.build();
     }
 
+    private String getFurtherDirections(FurtherDirections furtherDirections) {
```

Comment:
Does it make sense to make this a public method on the furtherDirections object itself?

Reply:
Could do, would mean less bloat in the service

Reply:
Could call it something like `getFurtherDirectionsText`